app/controller_services/shopping_lists_controller/destroy_service.rb

```diff
@@ -35,6 +39,14 @@ def shopping_list
       @shopping_list ||= user.shopping_lists.find(list_id)
     end
 
+    def aggregate_list
+      game.aggregate_shopping_list
+    end
+
+    def game
+      @game ||= shopping_list.game
```

Comment:
It bothers me a little bit defining this value in this way - it's a bit brittle because if `shopping_list` were to not be called until after the shopping list was deleted, this would raise an error. However, if that happened, the `#shopping_list` method would also raise `ActiveRecord::RecordNotFound`, so it wouldn't even get to the point of this being called.